### DISCUSSION TITLE
coupling terms not matching the equation in article

### DISCUSSION THREAD

**Discussion by One-and-Done2022** [dfki-ric/movement_primitives#54] [General]
![屏幕截图 2024-03-04 125111](https://github.com/dfki-ric/movement_primitives/assets/99159451/45b431fb-dc33-41bd-955e-1a8f50eaf2a3)
![屏幕截图 2024-03-04 125208](https://github.com/dfki-ric/movement_primitives/assets/99159451/350e1c49-73c4-47a5-9a7d-5b758aa5bc84)
why  F12 = self.k * (-self.desired_distance - da),should it be (self.desired_distance-da)

**Comment by AlexanderFabisch** [comment]
I am not entirely sure about this. It is working in this example though: https://github.com/dfki-ric/movement_primitives/blob/main/examples/plot_coupling_1d_to_1d.py
